### ISSUE TITLE
Get UNEXPLORED status after sometime,

### ISSUE THREAD

**Issue by omidziaee** [optimatika/ojAlgo#525]
I am running almost the same problem each 5 minutes, just some inputs to the problem changes. At first it runs smoothly and after a while it times out (I set the abort time to 100 seconds) and get the UNEXPLORED status. Is there a flag or something that I need to set at the end of each round to fully restart the solver? Any help would be appriciated.
Omid

**Comment by apete** [comment]
You have to give more details on how you do things. Are you using ExpressionsBasedModel? Which solver is used?

**Comment by omidziaee** [comment]
Yes I am using ExpressionsBasedModel and the default solver.

**Comment by omidziaee** [comment]
How do you set the solver? I assume ExpressionBasedModel selects the solver based on the problem. Am I wrong?

**Comment by apete** [comment]
Yes, `ExpressionBasedModel` selects an appropriate solver based on the problem type. There's a method `addIntegration(Integration<?>)` that allows you to add 3:d party solvers.
